bedrock/firefox/templatetags/helpers.py

```diff
@@ -173,6 +173,59 @@ def download_firefox(ctx, channel='release', platform='all',
     return jinja2.Markup(html)
 
 
+@library.global_function
+@jinja2.contextfunction
+def download_firefox_thanks(ctx, dom_id=None, locale=None, alt_copy=None, button_class=None,
+                            locale_in_transition=False, download_location=None):
+    """ Output a simple "download firefox" button that only points to /download/thanks/
+
+    :param ctx: context from calling template.
+    :param dom_id: Use this string as the id attr on the element.
+    :param locale: The locale of the download. Default to locale of request.
+    :param alt_copy: Specifies alternate copy to use for download buttons.
+    :param button_class: Classes to add to the download button, contains size mzp-t-xl by default
+    :param locale_in_transition: Include the page locale in transitional download link.
+    :param download_location: Specify the location of download button for
+            GA reporting: 'primary cta', 'nav', 'sub nav', or 'other'.
+    """
+
+    channel = 'release'
+    locale = locale or get_locale(ctx['request'])
+    funnelcake_id = ctx.get('funnelcake_id', False)
```

Comment:
How does this work? If a `funnelcake_id` is defined somewhere on the page which calls the button this picks that up?

Reply:
If there's a funnelcake param in the url (e.g. ?f=123) then that will get passed to the /thanks page (e.g. /download/thanks/?f=123). I'll add a comment to document what it does.

Reply:
So we'd still have to use the other button for a campaign page that has a funnel cake? (That's okay, I just want to understand).

Reply:
No, funnel cakes should work for both button helpers 🙂